conf/modules.config

```
@@ -475,11 +467,11 @@ process {
     withName: '.*ANNOTATE_SNV:ENSEMBLVEP' {
         ext.prefix = { "${vcf.simpleName}_rohann_vcfanno_filter_vep" }
```

```suggestion
        ext.prefix = { "${vcf.simpleName}_vep" }
```
We are not running rhocall and vcfanno in this pipeline

---

That's true, fixed